proto/buf/alpha/registry/v1alpha1/user.proto

```diff
@@ -127,6 +129,12 @@ message DeleteUserRequest {}
 
 message DeleteUserResponse {}
 
+message DeactivateUserRequest {
+  string id = 1;
```

Comment:
Do we not want this to be something easy like a `username`? The user ID is hardly used anywhere.

Reply:
`GetUser` and other not `By*` functions are all id across all our APIs

Reply:
Discussed offline, keeping to id for now.